src/main/java/no/entur/uttu/export/netex/producer/NetexObjectFactory.java

```diff
@@ -50,7 +50,7 @@ public class NetexObjectFactory {
   public static final String NSR_XMLNS = "NSR";
   public static final String NSR_XMLNSURL = "http://www.rutebanken.org/ns/nsr";
 
-  @Value("${netex.export.version:1.11:NO-NeTEx-networktimetable:1.3}")
+  @Value("${netex.export.version:1.15:NO-NeTEx-networktimetable:1.3}")
```

Comment:
should be "1.15:NO-NeTEx-networktimetable:1.5" (latest nordic profile version = 1.5)